e2e/_suites/kubernetes-autodiscover/features/filebeat.feature

```diff
@@ -0,0 +1,37 @@
+@kubernetes-autodiscover
+@filebeat
+Feature: Filebeat
+  Use Kubernetes autodiscover features in Filebeat to collect logs.
+
+Scenario: Logs collection from running pod
+  Given "filebeat" is running with "hints enabled"
+   When "a pod" is deployed
+   Then "filebeat" collects events with "kubernetes.pod.name:a-pod"
+
+Scenario: Logs collection from failing pod
+  Given "filebeat" is running with "hints enabled"
+   When "a failing pod" is deployed
+   Then "filebeat" collects events with "kubernetes.pod.name:a-failing-pod"
+
+# This scenario explicitly waits for 60 seconds before doing checks
+# to be sure that at least one job has been executed.
```

Comment:
Btw, @mdelapenya, I had added this comment explaining why there is a explicit wait here, because it also looked funny to me. Is this the best way to comment this? Does gherkin support comments at the step level?